content/terraform-enterprise/releases/v202507-1.md

```diff
@@ -19,6 +19,7 @@ customer should care about the change. More information on
 
 ## Known Issues
 
+1. (Updated 12/15/2025) Users may experience failures when using certain S3-compatible storage solutions. An AWS library upgrade in this release introduces authentication issues with some third-party storage providers. To prevent errors when running plans, applies, or accessing Terraform state files, upgrade your storage solution to a version compatible with AWS SDK for Go v2. For additional details, refer to [issue 2960](https://github.com/aws/aws-sdk-go-v2/discussions/2960) on the AWS SDK GitHub issues page.
```

Comment:
```suggestion
1. (Updated 12/15/2025) You may experience failures when using some S3-compatible storage solutions. This release includes an AWS library upgrade that introduces authentication issues with some third-party storage providers. To prevent errors when running plans, applies, or accessing Terraform state files, upgrade your storage solution to a version compatible with AWS SDK for Go v2. For additional details, refer to [issue 2960](https://github.com/aws/aws-sdk-go-v2/discussions/2960) on the AWS SDK GitHub issues page.
```